src/python/pants/util/osutil.py

```diff
@@ -52,12 +50,6 @@ def known_os_names():
   return reduce(set.union, OS_ALIASES.values())
 
 
-if PY3:
-  IntegerForPid = (int,)
```

Comment:
Open question: we could choose to keep this as a type alias. If so, we would want to tweak it to this:

```python
Pid = int
```

Type aliases are totally valid and used in the MyPy / type hint world, per https://www.python.org/dev/peps/pep-0484/#type-aliases. I chose to get rid of it here because it seemed to create indirection and resulted in additional imports, but totally feasible and valid to add back in this modified form if you all would like.

Reply:
I would love to keep it as a type alias, in particular because in the realm this is used integers mean two very different things: File descriptors and pids. So being explicit for which of the two this represents would help a lot.

However, until we typecheck the code that uses it, I'm not sure it's worth having, so I would settle for a `TODO(#issue)` for this one.

Reply:
I'll add back! It's easy to do, even without fully type checking the code.